lib.python/generate.py

```diff
@@ -67,7 +67,12 @@ def generate(function):
     print('    clean_params(data)')
   final_url = '"' + url.replace('{', '" + ').replace('}', ' + "') + '"'
   if method == 'GET':
-    print('    return self.request("' + method + '", ' + final_url + ', return_raw=return_raw)')
+    print('    extra=""')
+    for param in optional_params:
+      print("    if %s is not None:"%(param))
+      print('      extra += "?%s=" + %s'%(param, param))
```

Comment:
I think you need to urlformat the params : https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote